main.py

```
@@ -0,0 +1,70 @@
+#    tax: Optional[float] = None
+
+
+class Timeline(BaseModel):
+    data: Dict[str, int] = Field(
+        default_factory=dict, examples=[[{"2021-01-01": 1, "2021-01-02": 2}]]
+    )
+
+
+# Define an endpoint
+@app.get("/")
+def read_root():
```

Ruff (0.8.0)

Missing return type annotation for public function `read_root`

(ANN201)